insights/tools/query.py

```diff
@@ -130,23 +130,28 @@ def get_datasources():
 
 
 def matches(d, path):
+    def _simple_file(src):
```

Comment:
@xiangce - if you use "in" function afterwards, you don't have to define _simple_file function.

Reply:
@koalakangaroo - In `insights_archives` and `sos_archives`, the `simple_command` is conversed to a `simple_file`, e.g.

~~~python
# default.py
rabbitmq_queues = simple_command("/usr/sbin/rabbitmqctl list_queues name messages consumers auto_delete")
# insights_archive.py
rabbitmq_queues = simple_file("insights_commands/rabbitmqctl_list_queues_name_messages_consumers_auto_delete")
~~~

In normal, for these kinds of `simple_command`, the `insights-info` should only focus on the name (or the full-name includes path) of the command.
For example, it's a bit incomprehensible for `insights-info ss` returns a spec like `rabbitmq_queues`, just because the `simple_command` has a argument `messages`.
~~~python
❱ insights-info ss
insights.specs.default.DefaultSpecs.rabbitmq_queues
~~~

Here, the `_simple_file` is just trying to limit the source of the `in` operation of these `simple_command` to the name of the command only.